web/src/main/java/org/wso2/testgrid/web/api/ProductService.java

```diff
@@ -124,4 +139,60 @@ public Response getAllProductStatuses() {
         }
         return Response.status(Response.Status.OK).entity(list).build();
     }
+
+    /**
+     * This method is able to get the latest report of a given product from the remote storage and return.
+     * <p>
+     * <p> The report is returned as a html file<p/>
+     *
+     * @return latest report of querying product
+     */
+    @GET @Path("/reports") @Produces("application/html") public Response getProductReport(
```

Comment:
Shall we have the annotations in a separate line?

Reply:
+1 fixed with 751dd40f5a5e69ca20eef7775268e501116da992